stardict.cc

```diff
@@ -466,22 +466,20 @@ string StardictDictionary::handleResource( char type, char const * resource, siz
     {
       QString articleText = QString( "<div class=\"sdct_h\">" ) + QString::fromUtf8( resource, size ) + "</div>";
 
+      const char * srcRegexpStr = "(<\\s*(?:img|script)\\s[^>]*src\\s*=\\s*[\"']*)(?![^>]*(?:data|https?|ftp):)";
+      const char * hrefRegexpStr = "(<\\s*link\\s[^>]*href\\s*=\\s*[\"']*)(?![^>]*(?:data|https?|ftp):)";
```

Comment:
Both `QRegularExpression` and `QRegExp` constructors take `const QString &`. Please replace both `const char *` types with `const QString` to construct a `QString` from `const char *` two times less.